apps/meteor/server/models/raw/ModerationReports.ts

```diff
@@ -213,7 +226,7 @@ export class ModerationReportsRaw extends BaseRaw<IModerationReport> implements
 		return this.updateMany(query, update);
 	}
 
-	async hideReportsByUserId(userId: string, moderatorId: string, reason: string, action: string): Promise<UpdateResult | Document> {
+	async hideMessageReportsByUserId(userId: string, moderatorId: string, reason: string, action: string): Promise<UpdateResult | Document> {
```

Comment:
I would recommend we keep as it was before, and maybe extend it.

Why: From the main console, if Admin selects the action "Dismiss Reports" it does not matter if user reports or message reports we dismiss all of them at once. And we could further (if required) introduce an optional param `userReport?: true` which if passed, we process with hiding the user reports, else we only hide the message reports.

Reply:
Actually I just noticed this change wasn't done by me, but the explanation still stands.